bin/get-deps.sh

```diff
@@ -57,36 +37,21 @@ if ! [ -f $EVTDIR/nsqd -a -f $EVTDIR/nsqadmin ]; then
 		tar xz --strip=2 -C $EVTDIR/ '*/bin/*'
 fi
 
-if which docker; then
-	# Suppress errors for this call, as docker returns non-zero when it can't talk to the daemon
-	set +e
-	echo -n "  Found: " && docker version --format '{{.Client.Version}}' 2>/dev/null
-	set -e
-else
-	echo "=== Installing docker ==="
-	if [ -f /etc/centos-release ]; then
-		sudo yum -y install docker
-	else
-		sudo yum -y install docker --enablerepo=rhel-7-server-extras-rpms
-	fi
+if ! command -v docker &> /dev/null && ! command -v podman &> /dev/null; then
+	echo 'Cannot find `docker` nor `podman`. Perhaps:'
+	echo '  sudo yum install docker'
+	exit 1
```

Comment:
Since technically neither `docker` nor `podman` are required, should we remove this check?

For instance, if using this script from a CI server, I wouldn't want it to simply exit without the presence of either or these two tools, since I might be using something like `buildah` and `crio` only (and would still want to be able to grab all of the other dependencies, despite this check failing).

Reply:
Yep, will do.